code/script.js

```diff
@@ -1,9 +1,34 @@
 // A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
 function toggle() {
   this.classList.toggle("active")
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+document.getElementById("FAQ1").onclick = toggle
+document.getElementById("FAQ2").onclick = toggle
+document.getElementById("FAQ3").onclick = toggle
+document.getElementById("FAQ4").onclick = toggle
+document.getElementById("FAQ5").onclick = toggle
```

Comment:
Since you can only call the Id once you are repeating the same line of code (method) to apply to all of your accordion-questions. If you instead went for the method getElementsbyClassName you could decreased it to one line of code:

document.getElementsByClassName('questions').onclick = toggle :)